src/components/Card.js

```diff
@@ -23,13 +23,70 @@ export default class Card extends Component {
         twitterIcon: <i className="fab fa-twitter-square" />,
         close: <i className="fas fa-times-circle" /> 
       },
+      favorite: false
     };
   }
+
+  componentDidMount() {
+    this.checkFavorite();
+  }
+
+  checkFavorite = () => {
+    if (localStorage.getItem("favoritesList")){
+      let localFavoritesList = JSON.parse(localStorage.getItem("favoritesList")).map(object => object.name);
+      if (localFavoritesList.includes(this.props.cardData.name)) {
+        this.setState({
+          favorite: true
+        })
+      }
+    } 
+  }
+
+  handleFavorites = () => {
+    if (this.state.favorite) {
+      this.removeFromFavorites();
+    } else {
+      this.addToFavorites();
+    }
+    this.setState({
+      favorite: !this.state.favorite
+    });
+  }
+
+  removeFromFavorites = () => {
```

Comment:
Wow, great logic on here! Not sure how you figured this out so quickly! 🧠
I like how you're using the name as the index in order to remove the Object from LocalStorage

Reply:
@KevinKra please be sure to thoroughly review the Local Storage bits of this code!
It's really sound logic that could easily be implemented in other use cases 👍🏻